tests/conftest.py

```
@@ -1165,31 +1165,41 @@ def default_sc(admin_client):
+        client=admin_client, hco_ns_name=hco_namespace.name, **getattr(request, "param", {})
+    )
 
 
 @pytest.fixture(scope="module")
-def hyperconverged_resource_scope_module(admin_client, hco_namespace, installing_cnv):
+def hyperconverged_resource_scope_module(request, admin_client, hco_namespace, installing_cnv):
     if not installing_cnv:
-        return get_hyperconverged_resource(client=admin_client, hco_ns_name=hco_namespace.name)
+        return get_hyperconverged_resource(
+            client=admin_client, hco_ns_name=hco_namespace.name, **getattr(request, "param", {})
+        )
```

Ruff (0.15.12)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)